pandas/core/internals/blocks.py

```diff
@@ -2134,7 +2134,7 @@ def _na_value(self):
 
     @property
     def fill_value(self):
-        return tslibs.iNaT
+        return tslibs.NaT.asm8
```

Comment:
`iNaT` is the standard usage for this.

Reply:
`iNaT ` is an `int`, but here wants a `numpy.datetime64('NaT')`.